### PR TITLE
WIP testing with CentOS8

### PR DESCRIPTION
**Pull request by hardys**
This aims to remove the RDO repo dependency for both RHEL8 and CentOS8 - the packages we need from there have reduced since e.g vbmc is now containerized and the qemu-kvm changes needed are in the main el8 repos now.

The main TODO item is how we handle install of openstackclient but for now this is handled via pip, similar to what is already done on Ubuntu.

Marking this WIP as I've only tested on RHEL8 so far, @yrobla is planning to try with CentOS and provide feedback on any additional fixes needed.

### CODE REVIEW

File: centos_install_requirements.sh [hunk 1]
```diff
@@ -20,39 +20,34 @@ sudo yum -y update
...
-fi
 
 if [[ $DISTRO == "centos7" ]]; then
-    sudo yum -y install epel-release dnf --enablerepo=extras
+  sudo yum -y install epel-release dnf --enablerepo=extras
+  # Install tripleo-repos, used to get a more recent version of some packages on CentOS7
+  sudo dnf -y --repofrompath="current-tripleo,https://trunk.rdoproject.org/${DISTRO}-master/current-tripleo" install "python*-tripleo-repos" --nogpgcheck
+  sudo tripleo-repos current-tripleo
+  # There are some packages which are newer in the tripleo repos
+  sudo yum -y update
```
Comment (elfosardo) [inline centos_install_requirements.sh]: let's use dnf here

Comment (elfosardo) [inline centos_install_requirements.sh]: let's use dnf here

**Comment by hardys** [comment]
> are we missing the centos8 part in vm-setup/roles/packages_installation/tasks/main.yml ?

Yes good point, I thought @yrobla already pushed a fix for that but it looks like we are missing that, I'll rebase and update, thanks!

**Comment by hardys** [comment]
Replaced by https://github.com/metal3-io/metal3-dev-env/pull/223

**Pull request closed without merging**